### PR TITLE
k8s: populate start time field in pod

### PR DESCRIPTION
**Pull request by rithujohn191**
### Description
Currently the start time field in the pod proto object is not populated, this PR populates the start time field in the pod proto

### Testing Performed
unit test

### CODE REVIEW

File: backend/service/k8s/pods.go [hunk 1]
```diff
@@ -165,25 +167,28 @@ func (s *svc) checkExpectedObjectMetaFields(expectedObjectMetaFields *k8sapiv1.E
...
-	// TODO: There's a mismatch between the serialization of the timestamp here and what's expected
-	// on the frontend.
-	// var launch *timestamp.Timestamp
-	// if converted, err := ptypes.TimestampProto(k8spod.Status.StartTime.Time); err == nil {
-	// 	launch = converted
-	// }
+	var startTimestamp *timestamppb.Timestamp
+	startTime := k8spod.Status.StartTime
+	if startTime != nil {
+		startTimeProto, err := ptypes.TimestampProto(startTime.Time)
```
Comment (danielhochman) [inline backend/service/k8s/pods.go:173]: ptypes is deprecated, use `timestamppb.New(startTime.Time)`

Comment (rithujohn191) [reply backend/service/k8s/pods.go]: Done

**Comment by danielhochman** [comment]
were you able to test that the front-end handles the time correctly?

**Comment by rithujohn191** [comment]
> were you able to test that the front-end handles the time correctly?

Currently this value is not consumed by the frontend in Clutch directly (the only flow that calls describePod is https://clutch.lyft.net/k8s/pod/delete and it does not use the timestamp value)

**Comment by smonero** [comment]
> were you able to test that the front-end handles the time correctly?

I tried it in my list pods PR and it works using {pod.startTime?.toString()}

**Pull request closed without merging**